package.json

```diff
@@ -1,15 +1,48 @@
 {
-  "name": "code-exercise",
-  "version": "1.0.0",
-  "description": "This is a shopping cart coding exercise",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "canned -p 5000 ./server"
-  },
-  "author": "",
-  "license": "MIT",
-  "dependencies": {
-    "canned": "^0.3.12"
-  }
-}
+    "name": "shopping-app",
+    "version": "1.0.0",
+    "description": "This is a shopping cart coding exercise",
+    "main": "index.js",
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "compile:sass": "node-sass resources/sass/main.scss resources/css/style.css",
+        "start": "node ./bin/www",
+        "start:dev": "npm run compile:sass-dev"
+    },
+    "author": "Navin Tyagi",
+    "license": "MIT",
+    "devDependencies": {
+        "babel-preset-env": "^1.7.0",
+        "browser-sync-webpack-plugin": "^2.2.2",
+        "copy-webpack-plugin": "^5.0.2",
```

Comment:
Where is the webpack plugin ?

Reply:
removed unwanted modules from package.json